app/services/search/page_searcher.rb

```diff
@@ -12,7 +29,8 @@ def initialize(params)
   end
 
   def search
-    [*@queries].each do |search_type, query|
+    pp 'in search, queries are ', @queries.inspect, @queries.class
+    @queries.each do |search_type, query|
       case search_type
```

Comment:
You're expecting a string value, when search_type could be a symbol.

Call `to_s` on `search_type` to be safe:

`case search_type.to_s`